### PR TITLE
Improve information about transactional-update (bsc#1206155)

### PR DESCRIPTION
**Pull request by tbazant**
### Are there any relevant issues/feature requests?

* bsc#1206155

### CODE REVIEW

File: xml/concept-transactional-update.xml [hunk 1]
```diff
@@ -63,25 +63,67 @@
...
-        the other invocations of the command. After reboot, the most recently
-        created snapshot will be used as your new root file system, and it will
-        not include changes done in the previous snapshots.
+        If you use the <literal>--continue</literal> option of the
+        <command>transactional-update</command> command when performing changes
+        to the file system, all <filename>/etc</filename> directory layers
+        created by each separate run of
+        <command>transactional-update</command>, except for the one in the
+        newest snapshot, are synchronized to the <literal>lowerdir</literal>
+        (the <literal>lowerdir</literal> can have several mount points).
```
Comment (laenion) [inline xml/concept-transactional-update.xml:124]: Imho this is a bit difficult to understand. How about some thing like
> When using the <literal>--continue</literal> option and the new snapshot is a descendant of the current snapshot, then the <filename>/etc</filename> overlays of all the snapshots in between will be added as additional directories to the <literal>lowerdir</literal> (the <literal>lowerdir</literal> can have several mount points).

However I'm also wondering if this bit of information is important for the user at all...

Comment (tbazant) [inline xml/concept-transactional-update.xml:124]:
```suggestion
        When using the <option>--continue</option> option
        and the new snapshot is a descendant of the current snapshot,
        then the <filename>/etc</filename> overlays of all the snapshots
         in between will be added as additional directories to the
          <filename>lowerdir</filename> (the <filename>lowerdir</filename>
          can have several mount points).
```

Comment (tbazant) [reply xml/concept-transactional-update.xml]: applied :-)

File: xml/concept-transactional-update.xml [hunk 2]
```diff
@@ -19,12 +19,12 @@
   <section xml:id="what-is-transactional-update">
     <title>What are transactional updates?</title>
     <para>
-      The &alp; (&alpshort;) was designed to use a read-only root file system.
-      This means that after the deployment is complete, you are not able to
-      perform direct modifications to the root file system, for example, by
+      To keep the base operating system stable and consistent, the &alp;
+      (&alpshort;) uses a read-only root file system. Therefore you are not
+      able to perform direct changes to the root file system, for example, by
```
Comment (dariavladykina) [inline xml/concept-transactional-update.xml:24]:
```suggestion
      (&alpshort;) uses a read-only root file system. Therefore, you cannot
      perform direct changes to the root file system, for example, by
```

**Review by dariavladykina — COMMENTED** [review]
LGTM, just a minor nit. Thanks!

**Pull request merged**